basics/handy-stuff-strings.md

```diff
@@ -424,6 +424,8 @@ ValueError: could not convert string to float: 'hello'
     print(message, "!!!")
     print(message, "!!!")
     ```
+3. Make a program to take the input from the user and check if it is a palindrome.<br>
+   (Hint: Google how to reverse a string)
```

Comment:
I was going to add this on the answer page, but this might be a better place for it:

```suggestion
   (Hint: A string is a palindrome if it is the same when reversed. Google how to reverse a string.)
```